API/AddContact.php

```diff
@@ -0,0 +1,48 @@
+<?php
+	$inData = getRequestInfo();
+	
+    $firstName = $inData["firstName"];
+    $lastName = $inData["lastName"];
+    $email = $inData["email"];
+    $phone = $inData["phone"];
+
+    //connection to database
+	$servername = "localhost";
+    $username = "username";
+    $password = "password";
+    $dbname = "myDB";
+
+    // Create connection
+    $conn = new mysqli($servername, $username, $password, $dbname);
+	if ($conn->connect_error) 
+	{
+		returnWithError( $conn->connect_error );
+	} 
+	else
+	{
+		$stmt = $conn->prepare("INSERT into Contacts (ID, firstName, lastName, email, phone, userID) VALUES(NULL, ?, ?, ?, ?, NULL)");
```

Comment:
Should use DEFAULT for ID and $userID for UserID. All contacts are bound by UserID.
All SQL keywords should be capitalized (convention).
```suggestion
		$stmt = $conn->prepare("INSERT INTO Contacts (ID, firstName, lastName, email, phone, userID) VALUES(DEFAULT, ?, ?, ?, ?, ?)");
```